src/index.html

```diff
@@ -11,7 +11,201 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()">
+
+      <fieldset class="personal-information block">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">
+            Surname:
+
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              required
+              maxlength="20"
+              autocomplete="off">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="name">
+            Name:
+
+            <input type="text"
+            name="name"
+            id="name"
+            required
+            maxlength="15"
+            autocomplete="off">
```

Comment:
```suggestion
            <input
              type="text"
              name="name"
              id="name"
              required
              maxlength="15"
              autocomplete="off"
            >
```
Formatting. Pay attention to indentations and closing bracket